editor/components/inserter/no-blocks.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+function NoBlocks( props ) {
+	const { children } = props;
```

Comment:
Minor, but you can destructure the props directly on the previous line (the function args)